### DISCUSSION TITLE
Capitalize 2nd Word Of Column Title

### DISCUSSION THREAD

**Discussion by mr1716** [netbox-community/netbox#14772] [Help Wanted!]
Hey, I am writing a custom plugin using Netbox tables and columns. I have some columns named like “Hello Dolly,” which are showing up as “Hello dolly.” How would I get it to display “Hello Dolly” as the column title and which .py file would I put it in? Was thinking of verbose_name but unsure which file to put it into and which format. Haven’t had luck.

**Comment by jeremystretch** [comment]
Yep, [`verbose_name`](https://django-tables2.readthedocs.io/en/latest/pages/api-reference.html#django_tables2.columns.Column) is the way to go. By default, django-tables2 will just apply `.title()` to the field name and replace underscores with spaces.

**Reply by mr1716** [reply]
Thanks. Much appreciated! Looks like the issue was with how the values were being transferred into the container. But when it was, it worked. Much appreciated!
